README.md

```diff
@@ -74,65 +163,98 @@ slic init event-tickets
+2. `.env.slic.local` - this file doesn't exist by default, but if you wish to make overrides to the default, create it and add lines to your heart's content.
+3. `.env.slic.run` - this file is generated by `slic` and includes settings that are set by specific `slic` commands.
+
+### Xdebug and `slic`
+
+#### Available Commands
+
+##### `slic xdebug help`
+
+List the available Xdebug commands.
+
+##### `slic xdebug status`
 
-The `slic use [plugin]` sub-command sets which plugin codeception will point to for test running. If you don't pass a
-target, the current working directory will be tried.
+See if Xdebug is enabled or disabled, the host information, and the path mapping to add to your IDE.
 
-If you are unsure which plugins are available for use, you can execute `slic info`.
+Note that this command cannot be ran within `slic shell` because you've SSH'd into the Codeception container which has no knowledge of *slic*.
 
-If you don't remember which plugin you are currently using, you can run `slic using`.
+#### Setup IDE for Xdebug
 
-There are a few flavors of `slic use`:
+##### PhpStorm
 
-* `slic use` – Attempts to set the current working directory as the current `slic` target (codeception, composer, npm, etc commands will run against that plugin).
-* `slic use [plugin]` – Sets a specific plugin (regardless of the current working directory) as the current `slic` target.
-* `slic use [plugin]/common` – Sets a plugin as the current `slic` target to the `common/` directory of the plugin.
-* `slic using` – Tells you which plugin you are currently "using" (i.e. the last plugin on which you ran `slic use [plugin]`).
+Set your localhost plugin folder to map to `/var/www/html/wp-content/plugins`.
 
-**NOTE: you cannot `slic use [plugin]` on multiple plugins at once. The `slic` command relies on its `.env.slic.run` file
-to dictate which plugin it is pointing at.**
+Video walk-throughs for [PhpStorm](https://drive.google.com/file/d/1sD8djXgmYWCUDCm_1XZNRx_GBbotmmiB/view?usp=sharing) and [VSCode](https://drive.google.com/file/d/1519M2SRVgWVgTm0Px6UKfBjoQgxCR7Cp/view?usp=sharing).
 
-#### `slic run [testsuite]`
+Screenshot from PhpStorm's video:
 
-The `slic run [testsuite]` does precisely what you would expect. It runs the test suite against the plugin that is currently
-being targeted by `slic use [plugin]`. This command is essentially a `codecept run` command, so you can pass all of the
-typical Codeception arguments for `codecept run`.
+![PhpStorm XDebug settings](/docs/images/slic-Xdebug-PhpStorm.png "PhpStorm XDebug settings")
 
-### Killing Tests or Stopping `slic`
+#### Enable/Disable Xdebug
 
-If you find yourself wanting to bring down the containers–whether to save on resources, bail out of tests, etc–you can
-do so with the `slic down` sub-command. Running `slic-down` will shut down the containers regardless of what is being
-run with them.
+1. `slic xdebug on`
+1. `slic xdebug off`
+1. Within `slic shell`:
+    1. `slic xon`
```

Comment:
Food for thought: the `xon` and `xoff` aliases came from my personal preference, I wonder if it would make sense, now that `slic` is not depending on a container of mine anymore, to `xdebug-on` and `xdebug-off` that would leave no doubt about what they do.

Reply:
I really like the shortness of `xon` and `xoff`. When you drop into `slic shell`, the helper output indicating what those are is nice and helpful.